host-controller/src/main/java/org/jboss/as/host/controller/operations/NativeManagementAddHandler.java

```diff
@@ -81,6 +85,7 @@ protected void installServices(OperationContext context, NativeInterfaceCommonPo
         NativeManagementServices.installManagementWorkerService(serviceTarget, context.getServiceRegistry(false));
         ManagementRemotingServices.installDomainConnectorServices(serviceTarget, ManagementRemotingServices.MANAGEMENT_ENDPOINT,
                 nativeManagementInterfaceBinding, hostControllerInfo.getNativeManagementPort(), securityRealm, options);
+        return Arrays.asList(ManagementRemotingServices.MANAGEMENT_ENDPOINT, UndertowHttpManagementService.SERVICE_NAME);
```

Comment:
Sorry; I put the comment ^^^ in the wrong place; should be no the http add handler.